Tests/ComposableArchitectureTests/Support/ProcessInfo.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension ProcessInfo {
+  var isCI: Bool {
+    self.environment["CI"].map { Bool($0.lowercased()) ?? ($0 == "1") } ?? false
```

Comment:
According to GitHub, [`== "true"`](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) should probably be good enough, but doesn't hurt to generalize...